core/src/Baked/Theme/Default/DefaultThemeFeature.cs

```diff
@@ -108,12 +107,12 @@ public virtual void Configure(LayerConfigurator configurator)
 
                     dtc.Title = data.Label is not null ? l(data.Label) : null;
                     dtc.Exportable = true;
+                    dtc.Component.Data = Context.Parent(options: o =>
+                    {
+                        o.Prop = $"row.{c.Property.Name.Camelize()}";
+                    });
```

Comment:
maybe checking if component has data or not and setting only when data is null is more practical. this way any convention will set the component data without concerning its order.